app/templates/new-home-view.jade

```diff
@@ -5,36 +5,36 @@ mixin box
     if me.isAnonymous() == true
       h6#classroom-edition-header(data-i18n="new_home.classroom_edition")
       div
-        button.teacher-btn.btn.btn-primary.btn-lg.btn-block(data-i18n="new_home.im_a_teacher")
+        button.teacher-btn.btn.btn-primary.btn-lg.btn-block(data-event-action="Homepage Click Teacher Button CTA", data-i18n="new_home.im_a_teacher")
       div
-        a.btn.btn-forest.btn-lg.btn-block(href="/courses", data-i18n="new_home.im_a_student")
+        button.student-btn.btn.btn-forest.btn-lg.btn-block(data-i18n="new_home.im_a_student")
```

Comment:
If it navigates to another page, it should be an `a` tag, not a `button`. Buttons are for forms/causing things to change, and `a` tags are for navigating pages.

I would leave the `href` on as well, and if we need a handler for each link, do something like this:

```
  onClickStudentLink: (e) ->
    e.preventDefault()
    window.tracker?.trackEvent 'Homepage Click Student Button', category: 'Homepage', ['Mixpanel']
    application.router.navigate(e.currentTarget.getAttribute('href'), { trigger: true })
```

But I think there's already something somewhere that intercepts all clicks on `a` tags, and we can probably add a tracker hook in there to track things if `data-event-action` is present. I'd have to investigate.